docu/Concepts/Snippets/1-Blockchain/Connector/Dezentralität.md

```diff
@@ -0,0 +1,70 @@
+## Dezentralität
+
+Ich habe von Anfang an versucht Gradido so dezentral wie möglich zu entwerfen.
+Dezentral an der Basis. Kryptografie und Blockchain-Technologie sind meiner Meinung nach die besten Technologien dafür.
```

Comment:
Can gradido be implemented in a cryptographic provable way?

Reply:
Ja.
## Es kann kryptografisch überprüft werden, das das Regelwerk eingehalten wurde. 
- Es kann überprüft werden, das für ein Konto nur maximal 1000 Gradidos pro Monat geschöpft wurden.
- Es kann überprüft werden, das kein Konto für sich selbst schöpft. 
- Mit der geplanten Rechtesystem kann auch überprüft werden das nur die von der Community beauftragten Schöpfen dürfen. 
- Es kann überprüft werden, das keiner mehr Geld ausgibt als er hat.

## Was nicht kryptografisch überprüft werden kann
- Das jeder nur genau ein Schöpfungsfähiges Konto hat, das müssen die Menschen untereinander prüfen
- Das jemand tatsächlich für sein Aktives Grundeinkommen etwas für die Gemeinschaft getan hat, das kann nicht vom Computer geprüft werden, das müssen die Menschen machen